sinks/api/types.go

```diff
@@ -148,24 +153,35 @@ type SupportedStatMetric struct {
 	GetValue func(*cadvisor.ContainerSpec, *cadvisor.ContainerStats) []internalPoint
 }
 
-// Timeseries represents a single metric.
+// Timeseries represents a set of points for a series.
 type Timeseries struct {
-	Point            *Point
-	MetricDescriptor *MetricDescriptor
+	// The name of the series.
+	SeriesName string
+
+	// Precision of points timestamps.
+	TimePrecision TimePrecision
+
+	// The labels used by this series.
+	LabelKeys []string
```

Comment:
Is this for optimization? Why can't we use all the labels defined as part of the point?